### ISSUE TITLE
pkg server tracebacks with TypeError: string argument expected, got 'bytes'

### ISSUE THREAD

**Issue by Prominic-Build** [omniosorg/pkg5#266]
This seems to happen when looking at package information summery.

Seems to match the following bug in OpenIndiana: https://www.illumos.org/issues/12742
fixed by the following  commit: https://github.com/OpenIndiana/pkg5/pull/80/commits/8b58c72d7b415ac8c50ff23b5f67b9faa3ee0699

**Comment by oninoshiko** [comment]
Sorry, that should have been filed from this account.

**Comment by citrus-it** [comment]
@oninoshiko Thanks for the report. I took a look at the patch and it seems to be fixing a problem with showing the licence for a package where the licence has some UTF-8 characters, and I can see that OmniOS does have this issue, for example:

```
151030% pkg info --license bind

pkg: 'utf-8' codec can't decode byte 0xf6 in position 7395: invalid start byte. 
```

Is this the problem you are seeing? Can you confirm which version of OmniOS and give an example a command that you are trying?

Thanks,

Andy

**Comment by oninoshiko** [comment]
I'm on r151030. 
I run mirrors of core, extra, and an internal package server. Since updating the package server from r151028 to r151030 I'm seeing this on most packages when browsing the web interface. (it doesn't seem to affect our internal packages for some reason.)

What I'm seeing doesn't seem to be an issue with the pkg command, but with the pkg.depotd server.

**Comment by citrus-it** [comment]
Ok, thanks. It looks like this was already fixed in 8a2a56f0489b0685c0171899cfb917fc7851175a (a slightly different fix to the one in OpenIndiana) but that was not back ported to r151030. Ill raise a PR to get that done, as well as fixing the remaining issue.

**Comment by oninoshiko** [comment]
Just wanted to confirm that resolved my issue.
Thanks!
